use_cases/flask_azure.md

```diff
@@ -0,0 +1,172 @@
+## Deploying a simple SendGrid Hello Mail App on Microsoft Azure
+
+### Setting Up SendGrid & Azure Accounts
+
+**Step 1:** Create a SendGrid Free Account from [here](https://sendgrid.com/free?source=sendgrid-python)
+
+**Step 2:** Create a Microsoft Azure Account: [Free-Tier](https://azure.microsoft.com/en-in/free/) | [Student-Pack](https://azure.microsoft.com/en-in/free/students/)
+
+**Step 3:** Create your SendGrid API Key from [here](https://app.sendgrid.com/settings/api_keys)
+
+**Step 4:** Validate your Generated SendGrid API Key using
+```bash
+curl -i --request POST \
+--url https://api.sendgrid.com/v3/mail/send \
+--header 'Authorization: Bearer <YOUR_SENDGRID_API_KEY>' \
+--header 'Content-Type: application/json' \
+--data '{"personalizations": [{"to": [{"email": "<RECEIPENT_MAIL>"}]}],"from": {"email": "<SENDER_MAIL>"},"subject": "Is API Key Working","content": [{"type": "text/plain", "value": "Yes, It is!"}]}'
+```
+
+> **If Result gives *HTTP/1.1 202 Accepted*, You're good to move forward.**
+
+
+### Setting up the App
+
+**Step 1:** Get local copy of the project using `git clone https://github.com/jaykay12/demo-sendgrid-azure.git`
+
+**Step 2:** Create a virtualenv for the web app using `virtualenv venv`
+
+**Step 3:** Activate the virtualenv using `source venv/bin/activate`
+
+**Step 4:** Update your environment with SendGrid-API Key generated in Step 3 of Phase 1 using,
+```bash
+echo "export SENDGRID_API_KEY=<YOUR_API_KEY>" > sendgrid.env
+source ./sendgrid.env
+```
+
+**Step 5:** Install all the dependencies of the web-app using `pip install -r requirements.txt`
+
+**Step 6:** Check if the app is running locally using `FLASK_APP=application.py flask run`
+
+**Step 7:** Hit: `127.0.0.1:5000/` from browser & check if the simple mail sending is working or not.
+
+> **If the Mail is Sent suceesfully with *Response.code 202*, You're good to move ahead.**
+
+
+### Deploying the App on Microsoft Azure
+
+**Step 1:** Open Azure CLI using bash(preferabbly) or PowerShell
+
+![azure-cli](https://user-images.githubusercontent.com/13948542/46796235-cb3f8d00-cd69-11e8-9eed-120671532b9d.png)
```

Comment:
It might be better to include these images in this repo as part of this PR, so display of this file isn't dependent on external content.